api/backend/blueflood/blueflood.go

```diff
@@ -125,6 +129,10 @@ type sampler struct {
 	bucketSampler func([]float64) float64
 }
 
+// The amount of time before other resolutions become available
+// It's not "const" so that it can be mocked out for tests
+var availableOnlyFull = time.Hour * 4
```

Comment:
Is this specific to our setup, or does blueflood statically set the thresholds for when lower resolution data is available?